user-guide/Standard_Apps/Monitoring_Solutions/Ping_Monitoring/Tutorial_Apps_Ping_Monitoring_App.md

```diff
@@ -0,0 +1,107 @@
+---
+uid: Tutorial_Apps_Ping_Monitoring_App
+---
+
+# Getting started with the Ping Monitoring tool
+
+In this tutorial, you will learn how to use the Ping Monitoring tool for network monitoring and diagnostics.
+
+Expected duration: 15 minutes
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created using DataMiner version 10.4.8 and Ping Monitoring version 1.0.1-CU12.
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.9-CU2 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- Your dataminer.services [organization has been verified](xref:CloudConnectionVerification).
+
+## Overview
+
+- [Step 1: Deploy the Ping Monitoring package from the Catalog](#step-1-deploy-the-ping-monitoring-package-from-the-catalog)
+- [Step 2: Create a new group](#step-2-create-a-new-group)
+- [Step 3: Create a new destination](#step-3-create-a-new-destination)
+- [Step 4: Analyze the results](#step-4-analyze-the-results)
+- [Step 5: Filter and access destination details](#step-5-filter-and-access-destination-details)
+- [Step 6: Delete a destination](#step-6-delete-a-destination)
+
+## Step 1: Deploy the Ping Monitoring package from the Catalog
+
+1. Look up the [Ping Monitoring](https://catalog.dataminer.services/details/cb1bd962-97a5-461b-80fd-a62b3799de96) package in the DataMiner Catalog.
+
+1. click the *Deploy* button.
+
+   > [!TIP]
+   > See [Deploying a Catalog item](xref:Deploying_a_catalog_item).
+
+1. Go to the root page of your DataMiner System and check if the tool has been added:
+
+   ![DMS root page](~/user-guide/images/DMAroot-PingMonitoring.png)
+
+   If you do not know where to find the homepage URL, open <https://dataminer.services/> in a browser, and copy the URL or click the *Home* button.
+
+   ![dataminer.services](~/user-guide/images/dataminerservices-PingMonitoring.png)
+
+1. Open the Ping Monitoring tool.
+
+   This is what the application will look like:
+
+   ![Ping Monitoring UI](~/user-guide/images/brand_new_destinations-PingMonitoring.png)
+
+> [!TIP]
+> See also: [Ping Monitoring UI overview](xref:Ping_Monitoring_UI)
+
+## Step 2: Create a new group
+
+Groups are used to organize the destinations configured in the application. Destinations that belong to the same group share the same properties that differentiate them from other destinations.
+
+1. In the Ping Monitoring tool, go to the *Groups* page.
+
+1. In the header bar, click *Create Group*.
+
+   This will open a pop-up window.
+
+1. 
```

Comment:
In a tutorial, we're supposed to provide detailed step-by-step instructions with a clear example so users can get a feel for how something works. However, these details are currently missing in your tutorial. I've started adding more detailed instructions, but from this point onwards I'll need input from you. Could you add detailed instructions with what users should fill in so that they get a working example in their system?